client/my-sites/stats/activity-log/index.jsx

```diff
@@ -14,14 +15,269 @@ import { getSiteSlug, isJetpackSite } from 'state/sites/selectors';
+			'auto-updates for Plugins and we\'ll manage those for you',
+			user: null,
+			type: 'plugin_needs_update',
+			timestamp: 1485220539222
+		},
+		{
+			title: 'Akismet updated to version 3.2',
+			description: 'Akismet Plugin was successfully updated to its latest version: 3.2.',
+			user: null,
+			type: 'plugin_updated',
+			timestamp: 1485220539222
+		},
+		{
+			title: 'Twenty Eighteen Theme was activated',
+			description: 'TwentyEighteen Plugin was successfully activated.',
+			user: { ID: 123, name: 'Jane A', role: 'Admin' },
+			type: 'theme_switched',
+			timestamp: 1485220539222
+		},
+		{
+			title: 'Twenty Sixteen updated to version 1.0.1',
+			description: 'Twenty Sixteen Plugin was successfully updated to its latest version: 1.0.1.',
+			user: { ID: 123, name: 'Jane A', role: 'Admin' },
+			type: 'theme_updated',
+			timestamp: 1485220539222
+		},
+		{
+			title: 'Site updated to Professional Plan, Thank you',
+			description: 'Professional Plan was successfully purchased for your site and is valid until February 15, 2018.',
+			user: { ID: 123, name: 'Jane A', role: 'Admin' },
+			type: 'plan_updated',
+			timestamp: 1485220539222
+		},
+		{
+			title: 'Professional Plan Renewed for another month',
+			description: 'Professional Plan was renewed for another month and is valid until February 28, 2017',
+			user: { ID: 123, name: 'Jane A', role: 'Admin' },
+			type: 'plan_renewed',
+			timestamp: 1485220539222
+		},
+		{
+			title: 'Photon was activated',
+			description: 'Photon module was activated in your site. All your images will now be served through the ' +
+			'WordPress.com worldwide network.',
+			user: { ID: 123, name: 'Jane A', role: 'Admin' },
+			type: 'activate_jetpack_feature',
+			timestamp: 1485220539222
+		},
+		{
+			title: 'Custom CSS was deactivated',
+			description: 'Custom CSS module was deactivated.',
+			user: { ID: 123, name: 'Jane A', role: 'Admin' },
+			type: 'deactivate_jetpack_feature',
+			timestamp: 1485220539222
+		},
+		{
+			title: 'This is some really cool post',
+			description: '',
+			user: { ID: 123, name: 'Jane A', role: 'Admin' },
+			type: 'site_backed_up',
+			timestamp: 1485220539222
+		},
+		{
+			title: 'This is some really cool post',
+			description: '',
+			user: { ID: 123, name: 'Jane A', role: 'Admin' },
+			type: 'site_backed_up',
+			timestamp: 1485220539222
+		},
+		{
+			title: 'This is some really cool post',
+			description: '',
+			user: { ID: 123, name: 'Jane A', role: 'Admin' },
+			type: 'site_backed_up',
+			timestamp: 1485220539222
+		},
+		{
+			title: 'Jetpack updated to 4.5.1',
+			subTitle: 'Plugin Update',
+			description: '',
+			icon: 'plugins',
+			user: { ID: 123, name: 'Jane A', role: 'Admin' },
+			time: '4:32pm',
+			actionText: 'Undo',
+			timestamp: 1483351202400
+		},
+		{
+			title: 'Jetpack updated to 4.5.1',
+			subTitle: 'Plugin Activated',
```

Comment:
Hi! I've found a possible matching string that has already been translated **33** times:
`translate( 'Plugin activated.' )` *ES Score: 17*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/activity-log-components)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).